Final_Project_Billinis-AR.qmd

```diff
@@ -0,0 +1,132 @@
+---
+title: "Final_Project_GRShipping"
+author: "Alex Billinis"
+format: html
+editor: visual
+---
+
+## Quarto
```

Comment:
Why is all of this at the top of your paper? This document should be formatted with a title and introduction just like any other paper for a class.